Model/src/main/java/org/apidb/apicommon/model/userdataset/ISASimpleTypeHandler.java

```diff
@@ -0,0 +1,90 @@
+package org.apidb.apicommon.model.userdataset;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.gusdb.wdk.model.WdkModelException;
+import org.gusdb.wdk.model.user.dataset.*;
+
+import javax.sql.DataSource;
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class ISASimpleTypeHandler extends UserDatasetTypeHandler {
+  public final static String NAME = "ISASimple";
+  public final static String VERSION = "1.0";
+  public final static String DISPLAY = "ISA Simple";
+
+  private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
```

Comment:
Might want to use JsonUtil.Jackson here for future compatibility.  See: https://github.com/VEuPathDB/FgpUtil/blob/8a7b60a38c2ad7f3b98c6a36aac3a8978a84072c/Json/src/main/java/org/gusdb/fgputil/json/JsonUtil.java#L33